src/jsCallback.php

```diff
@@ -19,11 +23,44 @@ public function flatternArray($response)
         return $out;
     }
 
+    public function jsRender()
+    {
+        if (!$this->app) {
+            throw new Exception(['Call-back must be part of a RenderTree']);
+        }
+
+        return (new jQuery())->ajaxec([
+            'uri'        => $this->getURL(),
+            'uri_options'=> $this->args,
+            'confirm'    => $this->confirm,
+        ])->jsRender();
+    }
+
+    public function setConfirm($text = 'Are you sure?')
+    {
+        $this->confirm = $text;
+    }
+
     public function set($callback, $args = [])
     {
+        $this->args = [];
+
+        foreach ($args as $key=>$val) {
+            if (is_numeric($key)) {
+                $key = 'c'.$key;
+            }
+            $this->args[$key] = $val;
+        }
+
         return parent::set(function () use ($callback) {
             $chain = new jQuery(new jsExpression('this'));
-            $response = call_user_func($callback, $chain);
+
+            $values = [];
+            foreach ($this->args as $key=>$value) {
```

Comment:
to avoid `Avoid unused local variables such as '$value'` warning this line should be:
`foreach(array_keys($this->args) as $key) {`